techniques/systemSettings/userManagement/userManagement/9.1/userManagement.st

```diff
@@ -104,6 +110,11 @@ bundle agent check_usergroup_user_parameters_&RudderUniqueID&
         string => "-u ${usergroup_user_uid[${usergroup_user_index}]}",
             if => "!usermanagement_user_uid_empty_${usergroup_user_index}";
 
+      # define inner class prefix of secondary group management
+      "inner_classprefix_secondary_group_${usergroup_user_index}"
+        string => canonify("user_secondary_groups_${usergroup_user_login[${usergroup_user_index}]}"),
```

Comment:
This does not look like a class prefix but an old_class_prefix, is that really what we want to use?